src/main/java/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep.java

```diff
@@ -100,9 +100,11 @@ public void onStart(StepContext context) {
 
                     @Override
                     public void onSuccess(StepContext context, Object result) {
-                        if (context.hasBody()) {
+                        try {
                             CommitStatusUpdater.updateCommitStatus(run, getTaskListener(context), BuildState.success, name, step.builds, step.connection);
                             context.onSuccess(result);
+                        } catch (NullPointerException e) {
+                            e.getMessage();
```

Comment:
Why are you calling `getMessage` then discarding the result?

Either log the exception, or (better) prevent it from being thrown to begin with.